docs/usage.rst

```diff
@@ -12,6 +12,12 @@ Command Line Interface
 
 .. include:: snippets/cli.rst
 
+``--format`` supports the following keywords for replacement: ``filename``,
+``line``, ``definition``, ``message``, ``explanation``, and ``lines``. For
+example:
+
+.. code::
+    %(filename)s:%(line)3d: %(definition)s: %(message)s
```

Comment:
This command line argument is pretty complex and it deserves more and better examples - include the entire command line *and* the an example output.